src/main/cpp/auton/drivePrimitives/VisionDrivePrimitive.cpp

```diff
@@ -51,23 +51,22 @@ void VisionDrivePrimitive::Init(PrimitiveParams *params)
 
     Logger::GetLogger()->LogData(LOGGER_LEVEL::PRINT, m_ntName, "m_pipelineMode", m_pipelineMode);
 
-    m_dragonVision = DragonVision::GetDragonVision();
-    if (m_dragonVision != nullptr)
+    m_vision = DragonVision::GetDragonVision();
+    if (m_vision != nullptr)
         //  m_dragonVision->setPipeline(m_pipelineMode);
 
         if (m_chassis != nullptr)
         {
             switch (m_pipelineMode)
-            { /*
-             case DragonLimelight::PIPELINE_MODE::APRIL_TAG:
-                 m_headingOption = ChassisOptionEnums::HeadingOption::FACE_APRIL_TAG;
-                 break;
-             case DragonLimelight::PIPELINE_MODE::CONE:
-             case DragonCamera::PIPELINE::CUBE:
-                 m_headingOption = ChassisOptionEnums::HeadingOption::FACE_GAME_PIECE;
-                 break;
-             default:
-                 break;*/
+            {
+            case DragonCamera::PIPELINE::APRIL_TAG:
+                m_headingOption = ChassisOptionEnums::HeadingOption::FACE_APRIL_TAG;
```

Comment:
Need to merge; we have broken out the heading options to be more specific, so April tag has been removed.